packages/suite/src/components/guide/GuidePanel.tsx

```diff
@@ -9,39 +11,72 @@ import {
     GuideCategory,
     Feedback,
 } from '@guide-views';
+import { MODAL } from '@suite-actions/constants';
 
-const Wrapper = styled.div`
-    background: ${props => props.theme.BG_WHITE};
-    display: flex;
-    flex-direction: column;
+const GuideWrapper = styled.div<{ isModalOpen?: boolean }>`
+    z-index: 11;
```

Comment:
What about to put it in Z_INDEX in `/packages/components/src/config/variables.ts`?

Reply:
db5ed42